src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/ExtMethods.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+namespace Microsoft.Rest.ClientRuntime.Azure.TestFramework
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+
+    public static class ExtMethods
```

Comment:
Use the full name, and what these methods extend - in this case ConnectionStringDictionaryExtensions